### DISCUSSION TITLE
AddComponent 를 이용한 ZepetoScript 는 Destory로 파괴가 불가능 한가요?

### DISCUSSION THREAD

**Discussion by lokimve7** [naverz/zepeto-studio-kor#1023] [5. Q&A]
let tween : HJTween = info.target.AddComponent<HJTween>();
와 같이 컴포넌트를 붙이고 해당 컴포넌트를 지우고 싶은데

GameObject.Destroy(tween);
과 같이 하면 지워지지 않는데 다른 방법이 있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 
AddComponent()를 통해서 붙인 컴포넌트가 제페토스크립트인가요? 오브젝트인가요?

**Reply by lokimve7** [reply]
제페토 스크립트(타입스크립트) 입니다.

**Reply by Zepeto-TechEvangelist** [reply]
넵, 확인 감사합니다.
현재 말씀하신 것처럼 addComponent로 가져온 제페토스크립트는 destroy가 불가합니다. 
추후 지원할 수 있도록 하겠습니다.
감사합니다.

**Reply by lokimve7** [reply]
네 답변 감사합니다.
